dask/dataframe/io/utils.py

```diff
@@ -73,25 +73,23 @@ def _meta_from_dtypes(to_read_columns, file_dtypes, index_cols, column_index_nam
     -------
     meta : DataFrame
     """
-    meta = pd.DataFrame(
-        {c: pd.Series([], dtype=d) for (c, d) in file_dtypes.items()},
-        columns=to_read_columns,
-    )
-    df = meta[list(to_read_columns)]
-
-    if len(column_index_names) == 1:
-        df.columns.name = column_index_names[0]
-    if not index_cols:
-        return df
-    if not isinstance(index_cols, list):
-        index_cols = [index_cols]
-    df = df.set_index(index_cols)
-    # XXX: this means we can't roundtrip dataframes where the index names
-    # is actually __index_level_0__
-    if len(index_cols) == 1 and index_cols[0] == "__index_level_0__":
-        df.index.name = None
-
-    if len(column_index_names) > 1:
+    data = {
+        c: pd.Series([], dtype=file_dtypes.get(c, "int64")) for c in to_read_columns
```

Comment:
So, any column not in dtypes will be "int64" by default? Is there a specific case you are guarding for here (where a column name is missing from `file_dtypes`)?

Reply:
This is the default numeric index, when no index is specified

Reply:
Ah, okay. That's fair